### PR TITLE
used a switch in web keyboard input for better performance

### PR DESCRIPTION
**Pull request by MrRar**

### CODE REVIEW

File: Html/Index.html [hunk 1]
```diff
@@ -2541,67 +2541,70 @@
...
+			case 115: return GlKeys.F4;
+			case 116: return GlKeys.F5;
+			case 117: return GlKeys.F6;
+			case 118: return GlKeys.F7;
+			case 119: return GlKeys.F8;
+			case 120: return GlKeys.F9;
+			case 121: return GlKeys.F10;
+			case 122: return GlKeys.F11;
+			case 123: return GlKeys.F12;
+			defalut: return 0;
```
Comment (croxxx) [inline Html/Index.html]: This seems like a typo to me.

Comment (MrRar) [inline Html/Index.html]: @croxxx If your talking about key word `default`, it is to have code execute if none of the cases match the expression.
If your talking about `return 0;`, it is in the old code so I assume it belongs.
What exactly do you mean?

Comment (croxxx) [inline Html/Index.html]: I meant the incorrect spelling of `default`. `l` and `u` are switched.
Seems GitHub does not highlight the line I commented on. Sorry for the confusion.

**Comment by MrRar** [comment]
@croxxx Man I'm so dumb, I should have noticed that especially after you pointed it out. Anyway, it's fixed now.

**Comment by croxxx** [comment]
I doubt the change will do much for performance but I think it increases readability.

**Pull request merged**